packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -24,12 +35,30 @@ function createSuite<TArgs extends StressSuiteArguments>(
  */
 export interface StressSuiteArguments {
 	/**
-	 * Whether the current run is a stress run, which generally runs longer or more programatically generated tests.
+	 * Indicates the "stress level" of the tests. The number of test seeds and the timeout threshold
+	 * will be adjusted according to the selected mode.
+	 */
+	stressMode: StressMode;
+}
+
+/**
+ * @internal
+ */
+export enum StressMode {
+	/**
+	 * Different modes are planned for use in specific scenarios:
+	 * - Short Mode: Runs when stress is turned off, as part of the PR gate.
+	 * - Normal Mode: Runs when stress is turned on, also as part of the PR gate.
+	 * - Long Mode: Runs periodically in a separate pipeline for extensive and random testing.
```

Comment:
We don't actually gate PRs on any `normal` runs, just post-gate in CI. I would also avoid using mixed terminology in terms of stress being on or off--let's talk only in terms of the mode. So something like this
```suggestion
	 * - Normal Mode: Runs when stress is turned on, also as part of the PR gate.
```
```suggestion
	 * Different modes are planned for use in specific scenarios:
	 * - Short Mode: Runs as part of the PR gate. Uses deterministic seeds.
	 * - Normal Mode: Runs as part of the post-check-in CI process when there are changes to the affected package. Uses deterministic seeds. See tools/pipelines/test-dds-stress.yml for more information.
	 * - Long Mode: Uses non-deterministic seeds. We have plans to run this periodically in a separate pipeline for further random testing.
```